_includes/code/graphql.additional.py

```diff
@@ -1,37 +1,41 @@
-import weaviate
-import json
+# TEST DISABLED - sandbox needed
 
-client = weaviate.Client(
-    'https://edu-demo.weaviate.network',
-    auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
-)
+# import weaviate
+# import json
+
+# client = weaviate.Client(
+#     'https://edu-demo.weaviate.network',
+#     auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
+# )
```

Comment:
You can use this WCS instance:

```python
client = weaviate.connect_to_wcs(
    cluster_url="https://hha2nvjsruetknc5vxwrwa.c0.europe-west2.gcp.weaviate.cloud",
    auth_credentials=weaviate.AuthApiKey("nMZuw1z1zVtnjkXXOMGx9Ows7YWGsakItdus"),  # RO key
    headers={
        "X-OpenAI-Api-Key": os.getenv("OPENAI_APIKEY")
    }
)
```

So the top 9 lines could be replaced with:
```python
# START-ANY
import weaviate
import weaviate.classes as wvc
from weaviate.collections.classes.grpc import Sort
import os

client = weaviate.connect_to_local()

# END-ANY


# Actual client instantiation
client = weaviate.connect_to_wcs(
    cluster_url="https://hha2nvjsruetknc5vxwrwa.c0.europe-west2.gcp.weaviate.cloud",
    auth_credentials=weaviate.AuthApiKey("nMZuw1z1zVtnjkXXOMGx9Ows7YWGsakItdus"),
    headers={
        "X-OpenAI-Api-Key": os.getenv("OPENAI_APIKEY")
    }
)
```